src/components/WwwFrame/TheFooter.vue

```diff
@@ -347,6 +347,7 @@
 				<div class="tw-my-2">
 					<a
 						href="//www.facebook.com/kiva/"
+						class="tw-inline-flex"
```

Comment:
On hover, was seeing a blank space with an underline because of the new space added before the invisible span. Inline-flex takes care of that.